### PR TITLE
Dashboard qualifier navigation

### PR DESCRIPTION
**Pull request by jolheiser**
Continuing from #186, this adds navigation to the dashboard.

I know you mentioned a Pug formatting issue, if you want to let me know I can fix it in both spots for this PR since this will likely have the same issue. 🙂

### CODE REVIEW

**Comment by dzikoysk** [comment]
Thanks :) In the previous PR you've just missed indentation before `:to` property. It looks fine here :)

**Comment by jolheiser** [comment]
I added the spaces, but I noticed my IDE must use 2 spaces where the project tends to use 4. 
Would you like me to clean that up as well?

**Comment by dzikoysk** [comment]
Well, nevermind, I'll just format it later. Pug formatting is a little messy in this project due to lack of  pug-plugin in prettier.

**Comment by jolheiser** [comment]
Okay, sounds good. Any idea what happened with the docker check? Clicking "details" takes me to a 404. 😕

File: reposilite-frontend/src/views/Index.vue [hunk 1]
```diff
@@ -32,7 +32,7 @@
                           span.ml-2
                               span(v-for="(element, idx) in splitQualifier()")
                                   router-link(
-                                  :to="splitQualifier().slice(0, idx + 1).join('/')"
+                                    :to="splitQualifier().slice(0, idx + 1).join('/')"
```
Comment (jolheiser) [inline reposilite-frontend/src/views/Index.vue:35]:
```suggestion
                                      :to="splitQualifier().slice(0, idx + 1).join('/')"
```

File: reposilite-frontend/src/views/dashboard/Index.vue [hunk 2]
```diff
@@ -19,7 +19,12 @@
         .flex.justify-between.pb-6.pt-2.px-2
             h1.font-bold
                 | Index of
-                span.ml-2 {{ this.qualifier }}
+                span.ml-2
+                  span(v-for="(element, idx) in splitQualifier()")
+                    router-link(
+                      :to="'/dashboard' + splitQualifier().slice(0, idx + 1).join('/')"
+                    ) {{ element }}
+                    span /
```
Comment (jolheiser) [inline reposilite-frontend/src/views/dashboard/Index.vue:27]:
```suggestion
                span.ml-2
                    span(v-for="(element, idx) in splitQualifier()")
                        router-link(
                            :to="'/dashboard' + splitQualifier().slice(0, idx + 1).join('/')"
                        ) {{ element }}
                        span /
```

**Comment by jolheiser** [comment]
Okay, _now_ I'm done pushing. 😬

**Pull request merged**